packages/infinite-scroll/src/js/infinite-scroll.js

```diff
@@ -426,7 +426,7 @@
         // for infinite scroll, if there are pages upwards then never allow it to be at the zero position so the up button can be active
         if (scrollTop === 0 && grid.infiniteScroll.scrollUp) {
           // using pixels results in a relative scroll, hence we have to use percentage
```

Comment:
I wasn't around when this was implemented, but this comment here makes me think that they tried pixels at one point and gave up for some reason. I need to look through the history to find out why.

Reply:
Using pixels is giving a smooth scrolling experience. Did you get time to check the issue @mportuga

Reply:
Sorry, not yet, my regular job has been pretty taxing lately so I haven't had the energy to check on ui-grid much. I will try to prioritize it soon, we are about to be in between projects, so I should have more time starting next week.